libs/ui-lib/lib/ocm/components/clusterWizard/ClusterWizardFooter.tsx

```diff
@@ -100,7 +101,7 @@ const ClusterWizardFooter = ({
     <WizardFooter
       alerts={alertsSection}
       errors={errorsSection}
-      onCancel={onCancel || handleCancel}
+      onCancel={!isDissconnectedMode ? onCancel || handleCancel : undefined}
```

Comment:
nit: generally positively-formed conditions are easier to read :) 
```suggestion
      onCancel={sDissconnectedMode ? undefined : onCancel || handleCancel }
```